flamingo/models/build.py

```diff
@@ -18,6 +18,7 @@ class Build(EmbeddedDocument):
     deploy_tag: str = None
     post_build_commands: List[str] = field(default_factory=list)
     build_args: KeyValue = field(default_factory=dict)
+    build_machine_type: str = None
```

Comment:
`Optional[str]`

Reply:
To use `Optional`, there's some heavy work that needs to be done on `gcp-pilot` first.
It crashes when using `issubclass` and some other things